components/brave_wallet_ui/components/desktop/views/crypto/index.tsx

```diff
@@ -168,13 +170,34 @@ const CryptoView = (props: Props) => {
     onToggleAddModal()
   }
 
+  const selectAsset = (asset: TokenInfo) => {
+    if (asset) {
+      history.push(`${WalletRoutes.Portfolio}/${asset.symbol}`)
+    } else {
+      onSelectAsset(asset)
+      history.push(WalletRoutes.Portfolio)
+    }
+  }
```

Comment:
If `asset` can be null or undefined, shouldn't this be reflected in the type in the function arguments?

Reply:
Done!